docs/security/03_roles.md

```diff
@@ -0,0 +1,250 @@
+# Understanding Roles
+
+# Relevant Multisigs
```

Comment:
Should be 
```suggestion
## Relevant Multisigs
```

Since it's nested under `# Understanding Roles`

Reply:
fixed